client/signup/controller.js

```diff
@@ -245,6 +245,20 @@ export default {
 				experiment.variationName === 'treatment_scrambled';
 		}
 
+		// See: 1113-gh-Automattic/experimentation-platform for details.
+		const isGuidedFlow = flowName === 'guided';
```

Comment:
> flowName === 'guided'

I believe users from both the flows `flow: guided` and `flow: onboarding` should be eligible for the BigSky and TrailMap experiments. cc: @wongasy to confirm.

Reply:
We'll probably rename guided to onboarding once everything is ready. We'll have to decide that. But for now, `guided` should be treated as the main flow. It helps us test everything easily without touching the live flow.